package.json

```diff
@@ -11,6 +11,7 @@
     "react-dom": "^18.1.0",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
+    "redux": "^4.2.0",
```

Comment:
- [ ] Kindly add not only `redux` but also `react-redux` in `package.json`. The `react-redux` lets your React components read data from the Redux store.

Reply:
Okay, i will add the dependency. thank you very much @elyor-doniyorov